### ISSUE TITLE
EfficientNet: Improving Accuracy and Efficiency through AutoML and Model Scaling

### ISSUE THREAD

**Issue by sile** [sile/readings#70]
URL
-----

https://arxiv.org/abs/1905.11946

Abstract
----------

> Convolutional Neural Networks (ConvNets) are commonly developed at a fixed resource budget, and then scaled up for better accuracy if more resources are available. In this paper, we systematically study model scaling and identify that carefully balancing network depth, width, and resolution can lead to better performance. Based on this observation, we propose a new scaling method that uniformly scales all dimensions of depth/width/resolution using a simple yet highly effective compound coefficient. We demonstrate the effectiveness of this method on scaling up MobileNets and ResNet. 
To go even further, we use neural architecture search to design a new baseline network and scale it up to obtain a family of models, called EfficientNets, which achieve much better accuracy and efficiency than previous ConvNets. In particular, our EfficientNet-B7 achieves state-of-the-art 84.4% top-1 / 97.1% top-5 accuracy on ImageNet, while being 8.4x smaller and 6.1x faster on inference than the best existing ConvNet. Our EfficientNets also transfer well and achieve state-of-the-art accuracy on CIFAR-100 (91.7%), Flowers (98.8%), and 3 other transfer learning datasets, with an order of magnitude fewer parameters. 

Table of Contents
--------------------

- (1) Introduction
- (2) Related Work
  - ConvNet Accuracy
  - ConvNet Efifciency
  - Model Scaling
- (3) Compound Model Scaling
  - (3-1) Problem Formulation
  - (3-2) Scaling Dimensions
  - (3-3) Compound Scaling
- (4) EfficientNet Architecture
- (5) Experiments
  - (5-1) Scaling Up MobileNets and ResNets
  - (5-2) ImageNet Results for EfficientNet
  - (5-3) Transfer Learning Results for EfficientNet
- (6) Discussion
- (7) Conclusion

BibTeX
------

```bibtex
@article{tan2019efficientnet,
  title={EfficientNet: Rethinking Model Scaling for Convolutional Neural Networks},
  author={Tan, Mingxing and Le, Quoc V},
  journal={arXiv preprint arXiv:1905.11946},
  year={2019}
}
```

See Also
----------

- GitHub:  https: //github.com/tensorflow/tpu/tree/ master/models/official/efficientnet

**Comment by sile** [comment]
> **Ovservation 1 -** Scaling up any dimension of network width, depth, or resolution improves accuracy, but the accuracy gain diminishes for bigger models.

**Comment by sile** [comment]
> ** Observation 2 -** In order to pursue better accuracy and efﬁciency, it is critical to balance all dimensions of network width, depth, and resolution during ConvNet scaling.
